.changeset/slimy-donuts-sort.md

```diff
@@ -0,0 +1,9 @@
+---
+'@commercetools-uikit/view-switcher': patch
+---
+
+Update `ViewSwitcher.Button` component to allow using it using no text and just an icon.
+
+It's `children` prop is now optional.
```

Comment:
Nit: It is "Its" here.

Its (without an apostrophe) is a possessive pronoun, like his or her, for nouns that don't have a defined gender. In contrast, it's (with an apostrophe) is the shortened form, or contraction, of it is or it has.

Reply:
Sorry, I missed that one 🤦 

Updated here: 11fc8a45c24b60d79b8191e700c5e87999e5782f